flink-tools/src/main/java/io/pravega/flinktools/AppConfiguration.java

```diff
@@ -172,7 +172,7 @@ public StreamConfig(final String argName, final ParameterTool globalParams) {
             // Copy stream's scope to default scope.
             tempPravegaConfig = tempPravegaConfig.withDefaultScope(stream.getScope());
 
-            final String keycloakConfigBase64 = params.get("keycloak", "");
+            final String keycloakConfigBase64 = params.get(argPrefix + "keycloak", "");
```

Comment:
This shouldn't be needed. Line 162 copies parameters with argPrefix to non-prefixed parameter names. Is line 167 logging the parameter "keycloak"?

Reply:
@claudiofahey with the current code, if a keycloak param is added, the same keycloak is used for both input and output streams.

Reply:
You should be able to set `input-keycloak` and `output-keycloak`. Have you tried that?

Reply:
@claudiofahey I retested the original code with your suggestions and it worked. I m going to close this PR. My changes are not necessary.